File: Student.java
import java.util.*;
public class Student
{
    int a=10;
    void add(int a,int b)
    {
        System.out.println(a+b);
    }
    Student()
    {
        String fullName="Nithin";
        int rollNum= 34;
        double semPerentage = 95.9;
        String collegeName = "MVGR";
        int collegeCode = 33;
        System.out.println("Name :"+fullName+"\n"+"Roll no :"+rollNum+"\n"+"Sem percentage : "+semPerentage+"\n"+"College name : "+collegeName+"\n"+"College code : "+collegeCode);
    }
    public static void main(String args[])
    {
        Student n = new Student();
        System.out.println(n.a);
        n.add(12,13);
    }
}

File: Student.cpp
#include<iostream>
#include<string>
using namespace std;
class Student{
    private:
        string name;
        int rollnum;
        double sempercentage;
        string collagename;
        int collagecode;
    public:
        Student(string name,int rollnum,double sempercentage,string collagename,int collagecode){
            cout<<"constructor invoked: "<<endl;
            cout<<"Name of the student: "<<name<<endl<<"pin num: "<<rollnum<<endl<<"Sem percentage: "<<sempercentage<<endl<<"Collage Name :"<<collagename<<endl<<"Collage code :"<<collagecode<<endl;
        }
        ~Student(){
            cout<<"Destructor called student details has been deleted"<<endl;
        }
};
int main(){
    Student obj=Student("Raju",504,85,"MVGR",33);
    Student obj2("Santhosh",503,95,"MVGR",33);
}